modules/logging/pom.xml

```diff
@@ -51,6 +51,10 @@
       <artifactId>oasp4j-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter</artifactId>
```

Comment:
why do we need that for logging module? IMHO there is no need to couple this with spring.
Maybe just a test dependency and forgot to add the scope?